Editor/Window/ConnectToFleetInput.cs

```diff
@@ -0,0 +1,188 @@
+        {
+            await UpdateFleetMenu();
+            _fleetNameDropdownContainer.RegisterValueChangedCallback(_ =>
+                {
+                    _gameLiftPlugin.CurrentState.SelectedFleetIndex = _fleetNameDropdownContainer.index;
+                    var currentFleet = _fleetsList[_gameLiftPlugin.CurrentState.SelectedFleetIndex];
+                    _fleetIdText.text = currentFleet.FleetId;
+                    FleetId = currentFleet.FleetId;
+                    _gameLiftPlugin.CoreApi.PutSetting(SettingsKeys.SelectedFleetIndex,
+                        _gameLiftPlugin.CurrentState.SelectedFleetIndex.ToString());
+                }
+            );
+        }
+        
+        private async Task UpdateFleetMenu()
+        {
+            if (_gameLiftPlugin.GameLiftWrapper != null)
+            {
+                _fleetsList = await _requestAdapter.ListFleets();
+                s_fleetNameList.Clear();
+                _fleetsList.ForEach(fleet => s_fleetNameList.Add(fleet.Name));
+                _fleetNameDropdownContainer.choices = s_fleetNameList;
+            }
+        }
+        
+        private async void SetupBootMenu()
+        {
+            await SetupFleetMenu();
+            
+            if (_fleetsList.Count >= 1)
+            {
+                _fleetState = FleetStatus.Selecting;
+            }
+            _fleetNameDropdownContainer.index = _gameLiftPlugin.CurrentState.SelectedFleetIndex;
+            UpdateGUI();
+        }
+
+        protected sealed override void UpdateGUI()
+        {
+            switch (_fleetState)
+            {
+                case FleetStatus.Creating:
+                    Show(_fleetNameInput);
+                    Show(_cancelButton);
+                    Show(_fleetCreateFoldout);
+                    Hide(_fleetNameDropdownContainer);
+                    Hide(_fleetConnectFoldout);
+                    break;
+                case FleetStatus.CreatingInitial:
+                    Show(_fleetNameInput);
+                    Show(_fleetCreateFoldout);
+                    Hide(_cancelButton);
+                    Hide(_fleetConnectFoldout);
+                    break;
+                case FleetStatus.Selecting:
+                    Show(_fleetNameDropdownContainer);
+                    Show(_fleetConnectFoldout);
+                    Hide(_fleetNameInput);
+                    Hide(_fleetId);
+                    Hide(_fleetStatus);
+                    Hide(_fleetCreateFoldout);
+                    break;
+                case FleetStatus.Selected:
+                    Show(_fleetNameDropdownContainer);
+                    Show(_fleetId);
+                    Show(_fleetStatus);
+                    Show(_fleetConnectFoldout);
+                    Hide(_fleetNameInput);
+                    Hide(_fleetCreateFoldout);
+                    break;
+            }
+        }
+
+        public enum FleetStatus
+        {
+            CreatingInitial,
```

Comment:
Same comment as for RegisterComputeInput

Reply:
Changed to NotCreated